README.md

```diff
@@ -205,7 +205,7 @@ event, err := storage.ReplayEvent("d2a1f85a-delivery-id-123")
    ./tools/dev.sh
 
    # Customize webhook secret
-   GITHUB_WEBHOOK_SECRET=my-secret ./tools/dev.sh
+   HUBPROXY_WEBHOOK_SECRET=my-secret ./tools/dev.sh
```

Comment:
All our env vars start with the `HUBPROXY_` prefix now. So I think this case is good.

Reply:
Agreed.